docs/book/src/quick-start.md

```diff
@@ -57,6 +57,29 @@ cd $GOPATH/src/example
 kubebuilder init --domain my.domain
 ```
 
+If there is an error similar to `dial tcp x.x.x.x:443: i/o timeout`, you need to set go proxy.
+Go version >= 1.13
+
+```bash
+go env -w GO111MODULE=on
+go env -w GOPROXY="https://goproxy.io,direct"
+
+# Set environment variable allow bypassing the proxy for selected modules (optional)
+go env -w GOPRIVATE="*.corp.example.com"
+
+# Set environment variable allow bypassing the proxy for specified organizations (optional)
+go env -w GOPRIVATE="example.com/org_name"
+```
+
+Go version <= 1.12
+
+```bash
+# Enable the go modules feature
+export GO111MODULE="on"
+# Set the GOPROXY environment variable
+export GOPROXY="https://goproxy.io"
+```
```

Comment:
The project do not support go 1.12

Reply:
All right,It is more appropriate to the issue or FAQ